{"totalCount":6,"grade":"E","repositories":[{"url":"https://github.com/MTES-MCT/ecobalyse","vulnerabilityAlerts":{"totalCount":6,"nodes":[{"dismissedAt":"2023-04-24T12:12:18Z","createdAt":"2023-03-18T13:31:30Z","securityVulnerability":{"severity":"MODERATE","package":{"name":"request"},"advisory":{"identifiers":[{"type":"GHSA","value":"GHSA-p8p7-x288-28g6"},{"type":"CVE","value":"CVE-2023-28155"}],"references":[{"url":"https://nvd.nist.gov/vuln/detail/CVE-2023-28155"},{"url":"https://github.com/request/request/issues/3442"},{"url":"https://github.com/request/request/pull/3444"},{"url":"https://doyensec.com/resources/Doyensec_Advisory_RequestSSRF_Q12023.pdf"},{"url":"https://github.com/github/advisory-database/pull/2500"},{"url":"https://github.com/cypress-io/request/blob/master/lib/redirect.js#L116"},{"url":"https://github.com/request/request/blob/master/lib/redirect.js#L111"},{"url":"https://github.com/cypress-io/request/pull/28"},{"url":"https://github.com/cypress-io/request/commit/c5bcf21d40fb61feaff21a0e5a2b3934a440024f"},{"url":"https://github.com/cypress-io/request/releases/tag/v3.0.0"},{"url":"https://security.netapp.com/advisory/ntap-20230413-0007"},{"url":"https://github.com/advisories/GHSA-p8p7-x288-28g6"}]}}},{"dismissedAt":"2023-07-11T15:06:22Z","createdAt":"2023-07-08T19:28:19Z","securityVulnerability":{"severity":"MODERATE","package":{"name":"tough-cookie"},"advisory":{"identifiers":[{"type":"GHSA","value":"GHSA-72xf-g2v4-qvf3"},{"type":"CVE","value":"CVE-2023-26136"}],"references":[{"url":"https://nvd.nist.gov/vuln/detail/CVE-2023-26136"},{"url":"https://github.com/salesforce/tough-cookie/issues/282"},{"url":"https://github.com/salesforce/tough-cookie/commit/12d474791bb856004e858fdb1c47b7608d09cf6e"},{"url":"https://github.com/salesforce/tough-cookie/releases/tag/v4.1.3"},{"url":"https://security.snyk.io/vuln/SNYK-JS-TOUGHCOOKIE-5672873"},{"url":"https://lists.debian.org/debian-lts-announce/2023/07/msg00010.html"},{"url":"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/3HUE6ZR5SL73KHL7XUPAOEL6SB7HUDT2"},{"url":"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/6PVVPNSAGSDS63HQ74PJ7MZ3MU5IYNVZ"},{"url":"https://security.netapp.com/advisory/ntap-20240621-0006"},{"url":"https://github.com/advisories/GHSA-72xf-g2v4-qvf3"}]}}},{"dismissedAt":null,"createdAt":"2024-08-07T19:25:54Z","securityVulnerability":{"severity":"MODERATE","package":{"name":"Django"},"advisory":{"identifiers":[{"type":"GHSA","value":"GHSA-jh75-99hh-qvx9"},{"type":"CVE","value":"CVE-2024-41989"}],"references":[{"url":"https://nvd.nist.gov/vuln/detail/CVE-2024-41989"},{"url":"https://docs.djangoproject.com/en/dev/releases/security"},{"url":"https://www.djangoproject.com/weblog/2024/aug/06/security-releases"},{"url":"https://github.com/django/django/commit/27900fe56f3d3cabb4aeb6ccb82f92bab29073a8"},{"url":"https://github.com/django/django/commit/fc76660f589ac07e45e9cd34ccb8087aeb11904b"},{"url":"https://github.com/pypa/advisory-database/tree/main/vulns/django/PYSEC-2024-67.yaml"},{"url":"https://github.com/advisories/GHSA-jh75-99hh-qvx9"}]}}},{"dismissedAt":null,"createdAt":"2024-08-07T19:26:10Z","securityVulnerability":{"severity":"MODERATE","package":{"name":"Django"},"advisory":{"identifiers":[{"type":"GHSA","value":"GHSA-795c-9xpc-xw6g"},{"type":"CVE","value":"CVE-2024-41990"}],"references":[{"url":"https://nvd.nist.gov/vuln/detail/CVE-2024-41990"},{"url":"https://docs.djangoproject.com/en/dev/releases/security"},{"url":"https://www.djangoproject.com/weblog/2024/aug/06/security-releases"},{"url":"https://github.com/django/django/commit/7b7b909579c8311c140c89b8a9431bf537febf93"},{"url":"https://github.com/django/django/commit/d0a82e26a74940bf0c78204933c3bdd6a283eb88"},{"url":"https://github.com/pypa/advisory-database/tree/main/vulns/django/PYSEC-2024-68.yaml"},{"url":"https://github.com/advisories/GHSA-795c-9xpc-xw6g"}]}}},{"dismissedAt":null,"createdAt":"2024-08-07T19:26:15Z","securityVulnerability":{"severity":"CRITICAL","package":{"name":"Django"},"advisory":{"identifiers":[{"type":"GHSA","value":"GHSA-pv4p-cwwg-4rph"},{"type":"CVE","value":"CVE-2024-42005"}],"references":[{"url":"https://nvd.nist.gov/vuln/detail/CVE-2024-42005"},{"url":"https://docs.djangoproject.com/en/dev/releases/security"},{"url":"https://www.djangoproject.com/weblog/2024/aug/06/security-releases"},{"url":"https://github.com/django/django/commit/32ebcbf2e1fe3e5ba79a6554a167efce81f7422d"},{"url":"https://github.com/django/django/commit/f4af67b9b41e0f4c117a8741da3abbd1c869ab28"},{"url":"https://github.com/pypa/advisory-database/tree/main/vulns/django/PYSEC-2024-70.yaml"},{"url":"https://github.com/advisories/GHSA-pv4p-cwwg-4rph"}]}}},{"dismissedAt":null,"createdAt":"2024-08-07T19:27:34Z","securityVulnerability":{"severity":"MODERATE","package":{"name":"Django"},"advisory":{"identifiers":[{"type":"GHSA","value":"GHSA-r836-hh6v-rg5g"},{"type":"CVE","value":"CVE-2024-41991"}],"references":[{"url":"https://nvd.nist.gov/vuln/detail/CVE-2024-41991"},{"url":"https://docs.djangoproject.com/en/dev/releases/security"},{"url":"https://groups.google.com/forum/#%21forum/django-announce"},{"url":"https://www.djangoproject.com/weblog/2024/aug/06/security-releases"},{"url":"https://github.com/django/django/commit/523da8771bce321023f490f70d71a9e973ddc927"},{"url":"https://github.com/django/django/commit/efea1ef7e2190e3f77ca0651b5458297bc0f6a9f"},{"url":"https://github.com/pypa/advisory-database/tree/main/vulns/django/PYSEC-2024-69.yaml"},{"url":"https://github.com/advisories/GHSA-r836-hh6v-rg5g"}]}}}]},"grade":"E"}]}